tests/sentry/incidents/endpoints/test_organization_alert_rule_activations.py

```diff
@@ -0,0 +1,87 @@
+from datetime import timedelta
+from functools import cached_property
+
+import pytest
+from django.utils import timezone
+
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.alert_rule_activations import AlertRuleActivationsSerializer
+from sentry.incidents.models.alert_rule import AlertRuleMonitorType
+from sentry.testutils.abstract import Abstract
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.datetime import freeze_time
+from sentry.testutils.skips import requires_snuba
+
+pytestmark = [pytest.mark.sentry_metrics, requires_snuba]
+
+
+class AlertRuleActivationsBase(APITestCase):
+    __test__ = Abstract(__module__, __qualname__)
+
+    endpoint = "sentry-api-0-organization-alert-rule-activations"
+
+    def setUp(self):
+        self.create_team(organization=self.organization, members=[self.user])
+        self.login_as(self.user)
+
+    @cached_property
+    def organization(self):
+        return self.create_organization()
```

Comment:
Does it work if you just do `self.organization` without this (same for other cached properties)? I think `APITestCase` inherits from the generic testcase

Reply:
I imagine it should?

i'm just copying some of the existing tests in `test_organization_alert_rule_index` though 😬